src/components/Table/types.ts

```diff
@@ -0,0 +1,13 @@
+import { ReactNode } from "react"
+
+export interface WithID {
+  id: string
+}
+
+export interface ColumnConfig<Datum extends WithID> {
+  label: string | null
+  getValue?: (datum: Datum) => ReactNode
+  sortable?: boolean
+}
+
+export type ColumnConfigs<Datum extends WithID> = ColumnConfig<Datum>[]
```

Comment:
To me it looks like these 2 types belong to the table, I think having them in the Table file would improve readability, as one wouldn't have to jump to a different file to see what the type means.

Reply:
Yep, wondered about this—I think I tend to agree 👍